config_app/js/core-config-setup/config-setup-tool.html

```diff
@@ -73,7 +81,7 @@
             <tr>
               <td>TLS:</td>
               <td>
-                <select class="form-control" ng-model="mapped.TLS_SETTING">
+                <select ng-disabled="readonly('hostname')" class="form-control" ng-model="mapped.TLS_SETTING">
```

Comment:
named this `is_field_readonly`

Reply:
@josephschorr The other functions in this component use camel casing. Example:
```
$scope.removeOIDCProvider = function(provider)
```
Does it make more sense to follow that casing convention? Or do you still think I should go with
```
$scope.is_field_readonly = ...
```

Reply:
if you name the field `is-read-only`, then Angular will create it as isReadOnly on the scope

Reply:
and yeah, name the function `isFieldReadonly`